### PR TITLE
pythonPackages.djangorestframework-simplejwt: init at 4.4.0

### PR DESCRIPTION
**Pull request by arnoldfarkas**
###### Motivation for this change
Adding djangorestframework-simplejwt to Python packages.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/djangorestframework-simplejwt/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  
+  propagatedBuildInputs = [ django djangorestframework pyjwt ];
+  
+  doCheck = false; # Test raises django.core.exceptions.ImproperlyConfigured
+  
+  meta = with lib; {
+    description = "A minimal JSON Web Token authentication plugin for Django REST Framework";
+    homepage = "https://github.com/davesque/django-rest-framework-simplejwt";
+    license = licenses.mit;
+
```
Comment (jonringer) [inline pkgs/development/python-modules/djangorestframework-simplejwt/default.nix]:
```suggestion
```

File: pkgs/development/python-modules/djangorestframework-simplejwt/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  version = "4.4.0";
+  
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "c315be70aa12a5f5790c0ab9acd426c3a58eebea65a77d0893248c5144a5080c";
+  };
+  
+  propagatedBuildInputs = [ django djangorestframework pyjwt ];
+  
+  doCheck = false; # Test raises django.core.exceptions.ImproperlyConfigured
```
Comment (jonringer) [inline pkgs/development/python-modules/djangorestframework-simplejwt/default.nix]:
```suggestion
  # Test raises django.core.exceptions.ImproperlyConfigured
  doCheck = false;
```

File: pkgs/development/python-modules/djangorestframework-simplejwt/default.nix (hunk 1 above)
Comment (arnoldfarkas) [reply pkgs/development/python-modules/djangorestframework-simplejwt/default.nix]: Done

File: pkgs/development/python-modules/djangorestframework-simplejwt/default.nix (hunk 2 above)
Comment (arnoldfarkas) [reply pkgs/development/python-modules/djangorestframework-simplejwt/default.nix]: Done

**Review by jonringer — APPROVED** [review]
diff LGTM
commit LGTM

```
[4 built, 4 copied (50.9 MiB), 7.9 MiB DL]
https://github.com/NixOS/nixpkgs/pull/78250
2 package built:
python37Packages.djangorestframework-simplejwt python38Packages.djangorestframework-simplejwt
```

**Comment by jonringer** [comment]
@GrahamcOfBorg build python37Packages.djangorestframework-simplejwt python38Packages.djangorestframework-simplejwt

**Pull request merged**